packages/suite/src/actions/bluetooth/isBluetoothDeviceConnected.ts

```diff
@@ -1,4 +1,9 @@
 import { DesktopBluetoothDevice } from './DesktopBluetoothDevice';
 
+/**
+ * Desktop bluetooth device in any of these connection status types is considered connected.
+ * IMPORTANT: device regularly advertises when in `disconnected` and not paired state
+ * @param device
+ */
```

Comment:
I wrote the `isBluetoothDeviceConnected` so it's hypocritical of me to criticize it :smile: but in the meantime I got feedback from @marekrjpolak that it is weird that the function is called "is connected", but it includes _even other_ states than the `'connected'`.
So I though about what it means: is bluetooth device visible and did we exchange any sort of contact.
`isBluetoothDeviceVisible` or `isBluetoothDevicePresent` etc. would be more appropriate.

In any case I'd take that into consideration for the JSDoc:
```suggestion
/**
 * Desktop bluetooth device in any of these connection status types means that it is present, visible and at least some contact has been made, even if not yet "connected" in the full sense.
 * IMPORTANT: device regularly advertises when in `disconnected` and not paired state
 */
```
And IMO no need to duplicate parameter without explanation when it's typed 🤷